Dockerfile.arm

```diff
@@ -0,0 +1,321 @@
+# Base image with apex and transformer engine, but without NeMo or Megatron-LM.
+ARG BASE_IMAGE=nvcr.io/nvidia/pytorch:24.02-py3
+
+FROM rust:1.82.0 as rust-env
+
+RUN rustup set profile minimal && \
+    rustup install 1.82.0 && \
+    rustup target add aarch64-unknown-linux-gnu && \
+    rustup default 1.82.0
+
+FROM ${BASE_IMAGE} AS bionemo2-base
+
+# Install NeMo dependencies.
+ENV WORKDIR=/build
+WORKDIR ${WORKDIR}
+
+ARG MAX_JOBS=4
+ENV MAX_JOBS=${MAX_JOBS}
+
+# See NeMo readme for the latest tested versions of these libraries
+ENV TORCH_CUDA_ARCH_LIST=9.0
```

Comment:
It's recommended to set when building apex - saw it in the log messages. I think it enables certain optimizations. Since this is a GH200 specific image, saw no reason not to set it.